### ISSUE TITLE
VR Support

### ISSUE THREAD

**Issue by TheButlah** [bevyengine/bevy#1030]
**What problem does this solve or what need does it fill?**

As a graphics-curious rust developer, I've been in the market for a Rust game engine that has the ability to work with a VR headset. I understand bevy is early in its development, but it would be awesome if the seeds could be planted now to support VR in the future. 

**Describe the solution would you like?**

A way to get the game to display to a VR headset. Later, a way to support 6DoF movement, if doing so would not be achievable by the game developer without special support from Bevy.

**Describe the alternative(s) you've considered?**

One likely useful library in this endeavor is https://github.com/Ralith/openxrs, as it provides bindings to the OpenXR library.

**Additional context**
Disclaimer: I am neither a graphics developer nor a VR developer, so others would be more qualified to discuss this subject than I. However, I thought I would bring it up as I am passionate about VR and having VR support in Bevy would be awesome.

**Comment by amberkowalski** [comment]
Hi there! Currently there are a ton of layers between bevy and the graphics hardware that need to have VR support added to them, so VR support is likely not going to be added to bevy in the near future.  
However, there is an effort to add VR support to all of those layers, and once this is finished I'm planning on implementing VR support for bevy.

**Comment by Moxinilian** [comment]
Thanks for the issue! VR definitely is something of interest. I'm closing this issue as a duplicate of #115, feel free to comment if you believe it would better live on its own.
